Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +11,80 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            if (string.IsNullOrEmpty(message))
+            {
+                return message;
+            }
+
+            return TransformMessage(message, shift);
         }
 
         public static string Decrypt(string message, byte shift)
         {
-            return message;
+            if (string.IsNullOrEmpty(message))
+            {
+                return message;
+            }
+
+            return TransformMessage(message, shift * -1);
+        }
+
+        private static string TransformMessage(string message, int shift)
+        {
+            var cipheredMessage = new StringBuilder();
+
+            for (var i = 0; i < message.Length; i++)
+            {
+                cipheredMessage.Append(GetNewLetter(message[i], shift));
+            }
+
+            return cipheredMessage.ToString();
+        }
+
+        private static char GetNewLetter(char letter, int shift)
+        {
+            if (!isLetter(letter))
+            {
+                return letter;
+            }
+
+            var letterPosition = GetLetterPosition(letter);
+
+            var newLetterPosition = (letterPosition + shift) % 26;
+            if (newLetterPosition < 0)
+            {
+                newLetterPosition += 26;
+            }
+
+            return IsUpper(letter) ? (char)(newLetterPosition + 'A') : (char)(newLetterPosition + 'a');
+        }
+
+        private static bool isLetter(char letter)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'isLetter' to match pascal case naming rules, consider using 'IsLetter'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5020649)